meshflow/gui/dock_widget.py

```diff
@@ -281,6 +283,15 @@ def _update_profile_flow(self):
         self._plot.addItem(self._time_line)
         self.on_time_change()
 
+    def _datetime_to_timestamp(self, qdatetime: QDateTime) -> float:
+        try:
```

Comment:
Why not just only this change :

```suggestion
            return qdatetime.toPyDateTime().replace(tzinfo=timezone.utc).timestamp())
```

After all, in QGIS time is UTC

Reply:
Even simpler